Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -136,7 +136,7 @@ def set_data(self, data):
         self.set_input_summary()
 
         if data is not None and data.domain.class_var is None:
-            self.Error.data_error("Data has no target variable.\n"
+            self.Error.data_error("Data must have a (single) target variable.\n"
```

Comment:
Can we be more explicit here? As in, a different message if there's no target vs if there are too many targets?

Something like `'Data must have one, not %d target variables.\n'`

Reply:
Agree. I would put two different errors here. They require similar, but different actions (add / remove).